### PR TITLE
ENH use new sysroot

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf @jjhelmus I do not know what to do about this block in the conda_build_config.yaml

```
FINAL_CONDA_PYTHON_SYSCONFIGDATA_NAME:
  - _sysconfigdata_x86_64_conda_cos6_linux_gnu   # [linux64]
  - _sysconfigdata_i686_conda_cos6_linux_gnu   # [linux32]
  - _sysconfigdata_powerpc64le_conda_cos7_linux_gnu  # [ppc64le]
  - _sysconfigdata_aarch64_conda_cos7_linux_gnu  # [aarch64]
```

Do I need it to point to the new sysroot?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -125,6 +129,8 @@ outputs:
         - {{ pin_subpackage("gxx_" ~ ctng_target_platform) }}
         - {{ pin_subpackage("gfortran_" ~ ctng_target_platform) }}
         - {{ pin_subpackage("binutils_" ~ ctng_target_platform) }}
+      run:
+        - sysroot_{{ ctng_target_platform }}
```
Comment (isuruf) [inline recipe/meta.yaml:133]:
```suggestion
```

Comment (beckermr) [reply recipe/meta.yaml:133]: why?

Comment (isuruf) [reply recipe/meta.yaml:133]: This would clobber.

Comment (beckermr) [reply recipe/meta.yaml:133]: clobber what?

Comment (beckermr) [reply recipe/meta.yaml:133]: I don't think we package any version of gcc with the sysroot built by ctng anymore.

Comment (isuruf) [reply recipe/meta.yaml:133]: It would clobber the files. This package uses all the files from the compiler and is not supposed to have any depenencies.

Comment (beckermr) [reply recipe/meta.yaml:133]: ahhhh it copies out of host

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by isuruf** [comment]
For the python sysconfigdata thing, we can keep it as it is now. We should add a `-conda-` to `python-feedstock` as well and then fix it here.

**Comment by beckermr** [comment]
ok @isuruf, ready to merge?

**Comment by beckermr** [comment]
Not today but yes.

**Pull request merged**